pip-dep/requirements.txt

```diff
@@ -16,7 +16,7 @@ RestrictedPython~=5.0
 requests-toolbelt==0.9.1
 shaloop==0.2.1-alpha.11
 scipy~=1.4.1
-syft-proto~=0.5.2
+git+https://github.com/gmuraru/syft-proto.git@9293debf9490aaa701c6ae50bd41707ebbcfbf9c#egg=syft-proto
```

Comment:
TODO: After [this](https://github.com/OpenMined/syft-proto/pull/1450) gets merged